### DISCUSSION TITLE
How do I secure my Bun/Hono backend API's?

### DISCUSSION THREAD

**Discussion by thadkrugman** [clerk/javascript#1714] [Support request - DO NOT OPEN HERE] [answered]
Hey everyone,

I know this is a dumb question but how do I secure my backend API's with Clerk? I'm learning backend dev and I'm using Bun and Hono. I have Clerk working on my frontend Remix project and I need to make sure that the user sending the requests to the backend is signed in. I got this working easily with Node/Express but I'm not sure which Clerk SDK to use for a javascript backend that doesn't use Express.

**Comment by balazshevesi** [comment]
did you figure it out?

**Comment by thiskevinwang** [comment, marked as answer]
@MadThad762 check out the Clerk Hono middleware, and let us know if that works.
https://github.com/honojs/middleware/tree/main/packages/clerk-auth
